### PR TITLE
Feature/offcanvas user info

### PR DESCRIPTION
**Pull request by schweitzdesign**
trgbfzz1zl

Bump to 3.3.3
Treat the desktop user info the same as the mobile offcanvas menu.

## TODO (as separate efforts)
* Ensure long user names break-word as necessary (sstonebrook@forever.com bug)
* Refine hover stage for desktop user info offcanvas toggle (thinking we should make all of it or part of it color-primary for consistency)
* Revisit entrance and exit animations for offcanvas menus
* Signed out offcanvas menu has signup / login buttons at top instead of bottom (can most likely be resolved w/ a conditional class wherein the order property is bumped up )

### CODE REVIEW

File: app/assets/stylesheets/forever_style_guide/globals/_mixins.scss [hunk 1]
```diff
@@ -41,3 +41,41 @@ $color-nav-hover: color('secondary');
   border: 0;
   border-#{$direction}: $header_active_border_height solid $color;
 }
+
+
+// Shared nav mixin for offcanvas mobile nav or desktop user ingo
```
Comment (alexisBimatovs) [inline app/assets/stylesheets/forever_style_guide/globals/_mixins.scss]: what is ingo?

File: app/assets/stylesheets/forever_style_guide/modules/_nav-account_dropdown.scss [hunk 2]
```diff
@@ -65,35 +68,18 @@ $padding-small-horizontal: 3px;
   overflow-y: hidden;
   transition: opacity 0.15s ease-in-out;
 
+  @media(max-width: $grid-float-breakpoint) {
```
Comment (alexisBimatovs) [inline app/assets/stylesheets/forever_style_guide/modules/_nav-account_dropdown.scss]: missing space

File: app/assets/stylesheets/forever_style_guide/modules/_nav-mobile-menu.scss [hunk 3]
```diff
@@ -2,6 +2,30 @@ $nav-icon_bar-width: 22px;
...
+  @media (max-width: $grid-float-breakpoint) {
+    @include off-canvas-style;
+    display: flex;
+    flex-direction: column;
+  }
+}
+
+.navbar-user_info {
+  @media (max-width: $grid-float-breakpoint) {
+    max-height: none !important; // important overrids nested navbar bootstrap stuff
```
Comment (alexisBimatovs) [inline app/assets/stylesheets/forever_style_guide/modules/_nav-mobile-menu.scss]: super minor but spelling on override here

File: app/assets/stylesheets/forever_style_guide/globals/_mixins.scss (hunk 1 above)
Comment (schweitzdesign) [reply app/assets/stylesheets/forever_style_guide/globals/_mixins.scss]: info 😄

**Pull request merged**